### PR TITLE
Add cucumber features for browser tests

### PR DESCRIPTION
**Pull request by ielerol**
This adds feature files to be used for Cucumber tests for a set of existing functionality. 

## Additions

Feature files for 

-  filterable list controls on filterable pages
- the Wagtail admin for filter controls
- publishing event times 
- page preview module
- related posts module
- sidebar breakout module
- snippets

## Checklist

* [ ] PR has an informative and human-readable title
* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated
* [ ] Reviewers requested with the [Assignee tool](https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/) :arrow_right:

### CODE REVIEW

**Comment by sebworks** [comment]
This is awesome

**Comment by ascott1** [comment]
+1 awesome

File: test/browser_tests/cucumber/features/suites/default/filterable list controls.feature [hunk 1]
```diff
@@ -0,0 +1,113 @@
+Feature: Filterable list controls
+	As a visitor to cf.gov
```
Comment (anselmbradford) [inline test/browser_tests/cucumber/features/suites/default/filterable list controls.feature]: Should `cf.gov` be spelled out to `consumerfinance.gov`?

Comment (ielerol) [reply test/browser_tests/cucumber/features/suites/default/filterable list controls.feature]: maybe? the other feature files currently abbreviate to cf.gov and I was matching those. I'm fine with whatever works for the people who will be using these to run tests.

Comment (sebworks) [reply test/browser_tests/cucumber/features/suites/default/filterable list controls.feature]: Yeah, this matches the existing language. We can @anselmbradford in modify across the feature files if you think its's necessary.

Comment (anselmbradford) [reply test/browser_tests/cucumber/features/suites/default/filterable list controls.feature]: Sorry, this was a trivial comment, shouldn't hold anything up here.

**Pull request merged**